libs/ett/core/shell/src/lib/ett-shell.component.ts

```diff
@@ -36,26 +48,112 @@ const selector = 'ett-shell';
       ${selector} {
         display: block;
       }
+
+      ${selector}__toolbar {
+        width: 100%;
+        display: flex;
+      }
+
+      ${selector}__toolbar h1 {
+        flex: auto;
+      }
+
+      ${selector}__toolbar .menu {
+        align-self: flex-end;
+      }
```

Comment:
suggestion: It should also be possible to use nesting like so:
```suggestion
      ${selector}__toolbar {
        width: 100%;
        display: flex;
        
        h1 {
          flex: auto;
        }
        
        .menu {
          align-self: flex-end;
        }
      }
```